trompace/client/client_itemlist.py

```diff
@@ -378,7 +379,11 @@ def insert_listitem_itemlist(contributor: str, name: str, description: str,
     itemlist_obj = itemlist_obj[0]
     itemlist_id = itemlist_obj["identifier"]
     itemlist_elements = itemlist_obj["itemListElement"]
-    itemlist_elements = sorted(itemlist_elements, key=lambda k: k['position'])
+    # We can't guarantee the order that items come out of the CE, so explicitly sort them
+    try:
+        itemlist_elements = sorted(itemlist_elements, key=lambda k: k['position'])
+    except TypeError:
+        raise ValueError("A ListItem with position null is present. Check the ItemList before to proceed.")
```

Comment:
no problem for now, but in python this is `None`, not null. Sorry, I guess my comment was a bit confusing because I said null.